rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -195,9 +195,12 @@ s32 cellHddGameExitBroken()
 }
 
 
-s32 cellGameDataGetSizeKB()
+s32 cellGameDataGetSizeKB(vm::ptr<u32> size)
 {
-	fmt::throw_exception("Unimplemented" HERE);
+	if (size) {
+		*size = 0;
```

Comment:
?? this still isn't implemented! Finding that this is the cause of a failure in the future will require some voodoo skills, surely you should at least log out that this is a hack/stub?